spec/requests/purchases_requests_spec.rb

```diff
@@ -67,6 +67,12 @@
 
           expect(Purchase.last.amount_spent_in_cents).to eq 100_054
         end
+
+        it "storage location defaults to organizations storage location" do
+        purchase = create(:purchase)
+        get edit_purchase_path(@organization.to_param, purchase)
+        assert_select('select#purchase_storage_location_id option[selected]').first[storage_location.name]
```

Comment:
I'm not familiar with the `assert_select` method and not sure I understand what it's doing here specifically (the docs don't seem to indicate this usage). Can you match this selector with the way we do checks elsewhere?

Reply:
I've updated the tests to use `match` as is used elsewhere in the tests.